### ISSUE TITLE
Unresolved dependency

### ISSUE THREAD

**Issue by bantonsson** [sbt/sbt-conductr#101]
Hi,

I just added the 1.0.1 version of the plugin to my project and this is the result.

```
[info] Resolving org.fusesource.jansi#jansi;1.4 ...
[warn] 	::::::::::::::::::::::::::::::::::::::::::::::
[warn] 	::          UNRESOLVED DEPENDENCIES         ::
[warn] 	::::::::::::::::::::::::::::::::::::::::::::::
[warn] 	:: com.typesafe.akka#akka-contrib-extra_2.10;1.18.0: not found
[warn] 	::::::::::::::::::::::::::::::::::::::::::::::
[warn]
[warn] 	Note: Unresolved dependencies path:
[warn] 		com.typesafe.akka:akka-contrib-extra_2.10:1.18.0
[warn] 		  +- com.typesafe.conductr:sbt-conductr:1.0.1 (sbtVersion=0.13, scalaVersion=2.10) (/Users/bjorn/Source/typesafe/cinnamon/master-2/project/plugins.sbt#L9-10)
[warn] 		  +- default:master-2-build:0.1-SNAPSHOT (sbtVersion=0.13, scalaVersion=2.10)
```

I needed to add the akka-contrib-extra resolver, but it wasn't mentioned anywhere.

**Comment by huntc** [comment]
I wonder why this just started to be an issue... any ideas? I would have thought that it would have been an issue before...

**Comment by bantonsson** [comment]
Probably always needed to add the resolver but I hade started the sbt-conductr project for 1.0.0, and that put it in my cache :wink:

**Comment by huntc** [comment]
That'd make sense. I'll update the README.

**Comment by bantonsson** [comment]
Thanks

**Comment by huntc** [comment]
Fixed with 5cdb3134abe2773c4de0653490ae488cae760052
